### ISSUE TITLE
Reduce watchdog timeout

### ISSUE THREAD

**Issue by fnoble** [swift-nav/piksi_firmware#382]
Once #141 is resolved

**Comment by soulcmdc** [comment]
@fnoble Has this been done since #141 is closed now?

**Comment by fnoble** [comment]
@soulcmdc: yes, we could go ahead and do this now. I'd suggest maybe 2 seconds for the thread and 4 seconds for the hardware watchdog
